mlir/lib/Target/LLVMIR/LLVMImportInterface.cpp

```diff
@@ -37,8 +37,8 @@ LogicalResult mlir::LLVMImportInterface::convertUnregisteredIntrinsic(
     return failure();
 
   Type resultType = moduleImport.convertType(inst->getType());
-  auto op = builder.create<::mlir::LLVM::CallIntrinsicOp>(
-      moduleImport.translateLoc(inst->getDebugLoc()),
+  auto op = ::mlir::LLVM::CallIntrinsicOp::create(
```

Comment:
```suggestion
  auto op = CallIntrinsicOp::create(
```
optional: the namespace seems redundant